src/ui/style/global.module.css

```diff
@@ -39,6 +39,7 @@ body {
    */
   display: grid;
   --body-width: 425px;
+  --body-max-width: 800px;
```

Comment:
should we add this to popup mode too?
also login page layout is a bit broken on wide screen

Reply:
Popup can't be this wide